nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -39,6 +39,8 @@ let
     pipewire = recursiveUpdate (if useSessionManager then defaults.pipewire else defaults.minimal) cfg.config.pipewire;
     pipewire-pulse = recursiveUpdate defaults.pipewire-pulse cfg.config.pipewire-pulse;
   };
+
+  usedForAudio = cfg.pulse.enable or cfg.alsa.enable or cfg.jack.enable;
```

Comment:
This is equivalent to `cfg.pulse.enable`, since the attribute should always exist. Replace with `||`.

Reply:
Ugh. Tired. Syntax bad. Let me fix.

Reply:
Fixed.